server/src/internalClusterTest/java/org/opensearch/discovery/ClusterManagerDisruptionIT.java

```diff
@@ -91,7 +91,7 @@ public void testClusterManagerNodeGCs() throws Exception {
 
         logger.info("waiting for nodes to de-elect cluster-manager [{}]", oldClusterManagerNode);
         for (String node : oldNonClusterManagerNodesSet) {
-            assertDifferentMaster(node, oldClusterManagerNode);
+            assertDifferentClusterManager(node, oldClusterManagerNode);
```

Comment:
Same as above `NoMasterBlockService`, `BlockMasterServiceOnMaster`, `getMasterName`

Reply:
They are all public method or class,
https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/server/src/main/java/org/opensearch/cluster/coordination/NoMasterBlockService.java#L44
https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/test/framework/src/main/java/org/opensearch/test/disruption/BlockMasterServiceOnMaster.java#L46
https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java#L1948
I will rename them in a following PR to resolve issue https://github.com/opensearch-project/OpenSearch/issues/1684, and target for version 3.0.